### PR TITLE
risks, info-center, fix header

### PR DESCRIPTION
**Pull request by noyaossi**
1. I added a new page called "Risks" to header.ejs. In this page we see a risks table of all endpoints. In addition there is a button called "download csv report", which downloads a summary file of the risks. Another button called "view mitigation advices" which opens another "Mitigation Advices" page.
2. I added a new page called "information center" to header.ejs, with links to guides and recommendations.
3. I fixed the header- it is straight on all pages.

### CODE REVIEW

File: routes/management.js
Comment (tal-az) [inline routes/management.js:1]: Why add another router? I'm not familiarly with this sort of best practice but there's already router/router.js.

File: controllers/management.js
Comment (tal-az) [inline controllers/management.js:1]: Why not under controller agent? Seems relevant there

File: controllers/management.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const { removeHostById } = require('../services/fleet');
+
+exports.removeHost = async (req, res) => {
+    try {
+        const hostId = req.params.id;
+        const hostInfo = req.body; // Ensure the client sends the necessary host info in the request body
```
Comment (tal-az) [inline controllers/management.js]: Add input validation

File: controllers/management.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        console.log(`Received request to remove host with ID: ${hostId}`);
+        const result = await removeHostById(hostId, hostInfo);
+        res.status(200).json({ message: 'Host removed successfully', data: result });
```
Comment (tal-az) [inline controllers/management.js]: Should respond with "task sent successfully" for the request not to take such a long time to answer. In order to get response of success/failure another feature should be implemented, perhaps notifications. Look at some of Mazi's functions for reference

File: services/system.js [hunk 3]
```diff
@@ -44,24 +48,42 @@ async function executeRemoteCommand(sshCon, command) {
...
+    }
   }
+    
+  async function remoteUninstallFleetAgent(host_id, username, password = null, privateKey = null) {
+    const sshConn = await getSSHCon(host_id, username, password, privateKey);
+    try {
+        console.log(`Uninstalling Fleet agent on ${host_id}`);
+        let response;
+        if (password) {
+            response = await executeRemoteCommand(sshConn, `echo ${password} | sudo -S dpkg -r fleet-osquery`);
```
Comment (tal-az) [inline services/system.js]: How does fleet-osquery exists on host?

**Review by tal-az — COMMENTED** [review]
Looks good!! Left few comments for things that I think you should change/explain motive for. Haven't ran this in my environment yet, just read the code.

File: routes/management.js
Comment (noyaossi) [reply routes/management.js:1]: You're right. I deleted it.

File: controllers/management.js
Comment (noyaossi) [reply controllers/management.js:1]: I deleted controllers/management.js.

File: controllers/management.js (hunk 1 above)
Comment (noyaossi) [reply controllers/management.js]: I deleted all the changes related to "remove host", and I will open a new branch for it.

**Review by maziland — APPROVED** [review]

**Pull request merged**